packages/reactivity/src/effect.ts

```diff
@@ -47,13 +21,54 @@ export type DebuggerEventExtraInfo = {
 
 export let activeEffect: ReactiveEffect | undefined
 
-export const ITERATE_KEY = Symbol(__DEV__ ? 'iterate' : '')
-export const MAP_KEY_ITERATE_KEY = Symbol(__DEV__ ? 'Map key iterate' : '')
+let _FinalizationRegistry = getGlobalThis()
+  .FinalizationRegistry as typeof FinalizationRegistry
+
+if (!_FinalizationRegistry) {
+  _FinalizationRegistry = class FakeFinalizationRegistry {
+    register() {}
+    unregister() {}
+  } as any
+  if (__DEV__) {
+    console.warn(`FinalizationRegistry is not available in this environment.`)
+  }
+}
+
+const registry = new _FinalizationRegistry<WeakRef<ReactiveEffect>>(
```

Comment:
Maybe do not need to introduce the fake Finalization registration, only initialize the register when the implementation exists.

Reply:
Right, it can be simplified.